util/failed-report-resender/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+	java
+	id("org.springframework.boot") version "3.5.5"
+	id("io.spring.dependency-management") version "1.1.7"
+}
+
+group = "gov.cdc.prime.router"
+version = "0.0.1-SNAPSHOT"
+description = "Resend failed reports"
+
+java {
+	toolchain {
+		languageVersion = JavaLanguageVersion.of(17)
+	}
+}
+
+configurations {
+	compileOnly {
+		extendsFrom(configurations.annotationProcessor.get())
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation("org.springframework.boot:spring-boot-starter-web")
```

Comment:
Is there anything we can do to stop this from being scanned by Snyk? Since this is internal code and unreachable unless you pull down the repo and build this utility it would be nice if it doesn't show up on those reports. But maybe the entire /util/ directory is already skipped. @devopsmatt

Reply:
Yes.